extensions/2.0/Khronos/KHR_materials_diffuse_transmission/README.md

```diff
@@ -0,0 +1,241 @@
+* Bastian Sdorra, Dassault Systèmes [@bsdorra](https://github.com/bsdorra)
+* Ed Mackey, AGI [@emackey](https://twitter.com/emackey)
+
+TODO
+## Acknowledgments
+
+TODO
+
+## Status
+
+Experimental
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension adds a Lambertian diffuse transmission BSDF to the metallic-roughness material. Thin, dielectric objects like leaves or paper diffusely transmit incoming light to the opposite side of the surface. For optically thick media (volumes) with short scattering distances and therefore dense scattering behavior, a diffuse transmission lobe is a phenomenological plausible and cheap approximation.
+
+<figure style="text-align:center">
+<p float="left">
+<img src="figures/leaves.jpg" height="350" />
+<img src="figures/thin_translucent.jpg" height="350" />
+</p>
+<sub><figcaption><em>(Thin) translucent leaves/foliage</em></figcaption></sub>
+</figure>
+
+## Extending Materials
+
+The effect is activated by adding the `KHR_materials_diffuse_transmission` extension to any glTF material.
+
+```json
+{
+    "materials": [
+        {
+            "extensions": {
+                "KHR_materials_diffuse_transmission": {
+                    "diffuseTransmissionFactor": 0.25,
+                    "diffuseTransmissionTexture": {
+                      "index": 0
+                    },
+                    "diffuseTransmissionColorFactor": [
+                      1.0,
+                      0.9,
+                      0.85
+                    ]
+                }
+            }
+        }
+    ]
+}
+```
+
+|                                     | Type                                                                | Description                                                                                                                                                                                         | Required                 |
+| ----------------------------------- | ------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------ |
+| **diffuseTransmissionFactor**       | `number`                                                            | The percentage of reflected, non-specularly reflected light that is transmitted through the surface via the Lambertian diffuse transmission, i.e., the strength of the diffuse transmission effect. | No, default: `0`         |
+| **diffuseTransmissionTexture**      | [`textureInfo`](/specification/2.0/README.md#reference-textureInfo) | A texture that defines the strength of the diffuse transmission effect, stored in the alpha (A) channel. Will be multiplied by the diffuseTransmissionFactor.                                       | No                       |
```

Comment:
`A texture that defines the diffuse transmission factor, stored in the alpha (A) channel. Will be multiplied by the diffuseTransmissionFactor. `

Reply:
Remember our lingo in glTF is a little strange.  A "factor" is what GLSL would call a "uniform".  A factor is specifically a uniform value that can exist in the absence of a texture, or be multiplied by a texture.

So the actual parameter name here is simply `diffuseTransmission`.  It happens to be presented as a factor / texture pair, which is how all texturable glTF parameters are presented.